<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CubParametr</class>
 <widget class="QDialog" name="CubParametr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>350</height>
   </size>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>811</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="ok">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>773</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Длина ребра:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rebro">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Цвет:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="color">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Красный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Зелёный</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Синий</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Отображение:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Полигонами(в разработке)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Точками</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Координаты центральной точки:  </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>      X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rebro_2">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>-1001</number>
         </property>
         <property name="maximum">
          <number>1001</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>       Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rebro_3">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>-1001</number>
         </property>
         <property name="maximum">
          <number>1001</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>        Z:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="rebro_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>-1000</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="ok_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вычилить площадь и объём</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>270</y>
      <width>318</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Площадь поверхности = </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pl">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Объём фигуры = </string>
    </property>
   </widget>
   <widget class="QLabel" name="ob">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>155</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
